docs/aspnetcore-mvc.md

```diff
@@ -86,6 +86,57 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+
+## Configuration for cloud services or load balancers
+
+If you plan to deploy your application to a cloud service, you may need to do additional customization. Cloud services usually create similar environments in which applications run inside a Docker container without using HTTPS. The front host or load balancer that the user has direct access to may or may not use HTTPS by-default. 
+
+Cloud services are vary in configuration, but general rules are:
+
+* Make sure Load Balancer or front host uses HTTPS. 
+
+This may be the default or may require some amount of fiddling as in case of AWS.
+
+
+* Your Application should be aware of HTTPS scheme used by user despite the App itself is using HTTP. 
+
+[Microsoft.AspNetCore.Authentication.BuildRedirectUri](https://github.com/aspnet/Security/blob/26d27d871b7992022c082dc207e3d126e1d9d278/src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs#L117) function relies on request scheme and will not work properly when front host uses HTTPS but the application is running over HTTP. 
+
+Please check this [Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer) on how to configure your application to work in such environments.
+
+Azure environment with Front Door load balancer should work properly after forwarded headers configuration is added like so:
```

Comment:
I don't think **Front Door** should be capitalized here.  The term **front door load balancer** should be defined in the first paragraph of this section; you may want to reduce the number of names used for this (front host, load balancer, front door load balancer).  On line 92 above, introduce the names/concepts, define them clearly, then use one identifier in the remainder of the document.

```suggestion
Set forwarded headers as follows to properly configure your Azure load balancer.
```